frontend/src/components/EngagementView/Attendance.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import React from 'react'
+// import type { Student } from 'flow/types'
+import { Entry, Panel } from 'components'
+
+type Props = {
+    noShows: number
+}
+
+const Attendance = (props: Props) => {
+    return (
+        <Panel header='Attendance'>
+            <Entry>No-shows: {props.noShows}</Entry>
```

Comment:
For consistency with rest of React code, would be good to use destructuring here. 
In line before return statement, add `const {noShows} = props`